tensorflow/python/eager/function.py

```diff
@@ -2940,6 +2942,11 @@ def flat_input_signature(self):
     """Returns the flattened input signature."""
     return self._function_spec.flat_input_signature
 
+  @property
+  def hashable_input_signature(self):
```

Comment:
afaict, this abstraction doesn't provide much benefit and adds an unnecessary indirection perf cost.  Let's just use `self._hashable_input_signature` directly.

Reply:
Done